modules/ossm-about-deploying-istio-using-sail-operator.adoc

```diff
@@ -0,0 +1,8 @@
+:_mod-docs-content-type: Concept
+[id="ossm-about-deploying-istio-using-sail-operator"]
+= About deploying Istio using the Sail Operator
+:context: ossm-about-deploying-istio-using-sail-operator
+
+To deploy {istio} using the Sail Operator, you must create an {istio} resource. Then, the Operator creates an `IstioRevision` resource, which represents one revision of the {istio} control plane. From the `IstioRevision` resource, the Operator deploys the {istio} control plane, which includes the `istiod` deployment and other resources.
+
+The Sail Operator may create additional instances of the `IstioRevision` resource, depending on the strategy defined in the {istio} resource.
```

Comment:
@luksa at one time we discussed how there is a deployment strategy and an update strategy. In this particular context, I think it seems to make sense to say upgrade strategy, but I just wanted to check in with you in case you think of something I'm not thinking of.

Reply:
discussed and made change.